assets/tailwind.config.js

```diff
@@ -0,0 +1,17 @@
+const colors = require('tailwindcss/colors')
+
+module.exports = {
+  purge: [],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {
+      colors: {
+        teal: colors.teal
```

Comment:
teal color doesn't come enabled by default in tailwindcss v2, so it's needs explicit enabling.